Leanplum-SDK/Classes/LPRequestStorage.m

```diff
@@ -34,54 +34,115 @@ - (id)init
 {
     if (self = [super init]) {
         _defaults = [NSUserDefaults standardUserDefaults];
```

Comment:
maybe we remove this here too, to avoid sb using it in the future? it seems only used in migrate method